P5/Source/Specs/model.identInfo.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Copyright TEI Consortium. 
+Dual-licensed under CC-by and BSD2 licences 
+See the file COPYING.txt for details
+$Date$
+$Id$
+-->
+<?xml-model href="https://jenkins.tei-c.org/job/TEIP5-dev/lastSuccessfulBuild/artifact/P5/release/xml/tei/odd/p5.nvdl" type="application/xml" schematypens="http://purl.oclc.org/dsdl/nvdl/ns/structure/1.0"?>
+<classSpec xmlns="http://www.tei-c.org/ns/1.0" module="tei" xml:id="IDENTINFO" type="model" ident="model.identInfo">
+  <desc versionDate="2022-09-09" xml:lang="en">groups elements which provide an explanation or an equivalency for a markup construct.</desc>
```

Comment:
@sydb I'm not sure I understand why we need this model.identInfo. The name is very easy to confuse with the larger model of which this is a subset, model.identInformation. What about model.identEquiv instead?

Reply:
Well we certainly need 2 model classes, here. One is used by 6 elements, the other by 7 elements:
~~~
model.identInfo ↢ model modelGrp modelSequence moduleSpec paramSpec schemaSpec
model.identInformation ↢ attDef classSpec constraintSpec dataSpec elementSpec macroSpec valItem
~~~
But I could easily be convinced those are not particularly good names for these classes. I am convinced they are better than model.glossLike. But I am not remotely convinced we can’t do better. They are really just what jumped to mind.
I am not particularly fond of model.identEquiv, but I guess if you think of `<equiv>` as a formal XSLT-processable equivalent, and `<gloss>` as the natural language equivalent it is workable. Would you keep the larger class (`<gloss>`, `<equiv>`, `<altIdent>`) as model.identInformation, or change it to model.identEquivOrAlternate or something else?

Reply:
I think the larger class could be simplified as model.identInfo, if we make the smaller class model.identEquiv.